management/src/clusterctl/main.go

```diff
@@ -179,7 +196,11 @@ func newPostActioner(validateCb validateCallback, postCb postCallback) *postActi
 }
 
 func (npa *postActioner) procFlags(c *cli.Context) {
-	npa.extraVars = c.String("extra-vars")
+	npa.flags.extraVars = c.String("extra-vars")
```

Comment:
may be we should use `extraVarsFlag.Name` instead of `"extra-vars"`. We can also add variable for host-group flag.